### ISSUE TITLE
[Manage attachments] Several MIME type are not supported, e.g. audio/aac, application/x-cdf, audio/opus, audio/x-midi

### ISSUE THREAD

**Issue by VolhaBazhkova** [epam/ai-dial-chat#1370]
### EPAM AI DIAL chat version

0.10.0

### What happened?

**Reproduced the issue locally:** 
1. In DIAL code set the following parameter like allowedTypes={['audio/aac']}
2. Click on Clip icon on conversations panel
3. Note that on manage attachments window there is "Supported types: no available extensions" -> **bug**
4. Error if to load .aac file type
![image](https://github.com/epam/ai-dial-chat/assets/111180458/020ed16f-563f-412f-a449-b88efa434710)

Seems that the issue is also reproduced for
- application/x-cdf
- audio/opus
- audio/x-midi (note that audio/midi works fine)

Note: audio/mpeg works fine

**Note:** In the verifications this table with MIME types was used: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

### Confidential information

- [X] I confirm that do not share any confidential information

**Comment by denys-kolomiitsev** [comment]
Could be closed as chat accepts all mentioned MIME types.

**Comment by VolhaBazhkova** [comment]
@denys-kolomiitsev I agree that DIAL Chat accepts and works with .acc files (and others) correctly in the apps/models where "Supported types: all."
The issue is that if user sets the exact type, File manager doesn't allow the files with the extension to be loaded. Please see the details on the screenshot below:

<img width="1866" height="926" alt="Image" src="https://github.com/user-attachments/assets/3dd2caf1-a74b-4633-b6b2-26375ab08b40" />

**Comment by VolhaBazhkova** [comment]
Discussed with Denys and re-checked: File Manager supports .acc files with correct MIME type, for example, "audio/vnd.dlna.adts". It's better to configure MIME type: "audio/*"
